packages/plugins/color-picker/admin/src/components/tests/color-picker-input.test.js

```diff
@@ -25,6 +25,21 @@ const App = (
 );
 
 describe('<ColorPickerInput />', () => {
+  /**
+   * We do this because –
+   * https://github.com/facebook/jest/issues/12670
+   */
+  beforeAll(() => {
+    jest.setTimeout(30000);
```

Comment:
we had a similar issue with Alex no long ago, we discussed the possibility to increase the time out overall because it often happens 🤷🏻‍♀️

Reply:
Jest is so flaky, yet no one has brought out a viable alternative – maybe vitest, but that's not exactly been around that long.